integrations/dd-sdk-android-cronet/src/test/kotlin/com/datadog/android/cronet/internal/CronetRequestResourceIdentifierTest.kt

```diff
@@ -0,0 +1,193 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.cronet.internal
+
+import com.datadog.android.core.internal.net.HttpSpec
+import com.datadog.android.rum.internal.net.RumResourceInstrumentation.Companion.buildResourceId
+import com.datadog.android.utils.forge.Configurator
+import fr.xgouchet.elmyr.annotation.StringForgery
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.chromium.net.UploadDataProvider
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.mock
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@ForgeConfiguration(Configurator::class)
+internal class CronetRequestResourceIdentifierTest {
+
+    @StringForgery(regex = "http(s?)://[a-z]+\\.com/\\w+")
+    private lateinit var fakeUrl: String
+
+    @StringForgery(regex = "x-[a-z]+/[a-z]+")
+    private lateinit var fakeContentType: String
+
+    @StringForgery
+    private lateinit var fakeBody: String
+
+    @Test
+    fun `identify GET request`() {
```

Comment:
nit: here and below it is better to add `Given/When/Then` and follow test naming convention. It was like that before, but we can take an opportunity to update it a bit.